src/main/java/peoplesoft/commons/core/index/Index.java

```diff
@@ -9,6 +9,8 @@
  * convert it back to an int if the index will not be passed to a different component again.
  */
 public class Index {
+    public static final String MESSAGE_CONSTRAINTS = "Index should be a non-zero positive integer.";
+
```

Comment:
If the user will see this message, I really think HR managers want to put this kinda lingo behind them 😩 If not, it is fine.